src/main.c

```diff
@@ -67,6 +68,8 @@ static void usage(void)
 		"  -s, --source\t\t\tRun in \"source mode\" to scan a policy source repository\n"\
 		"\t\t\t\tthat is designed to compile into a full system policy.\n"\
 		"  -S, --summary\t\t\tDisplay a summary of issues found after running the analysis\n"\
+		"      --summary-only\t\tOnly display a summary of issues found after running the analysis\n"\
```

Comment:
Can you add the period at the end of this line as well?  It makes the man page output look weird without it.